src/JsonMapFileParser.cc

```diff
@@ -113,6 +115,31 @@ namespace ChimeraTK::detail {
     size_t numberOfElements{1};
     size_t bytesPerElement{0};
 
+    struct DoubleBufferingInfo {
+      struct Address {
```

Comment:
There is a struct Address below which is different. Something is not right here.

Reply:
Ah, this is in the DoubleBufferingInfo, the other is outside. Are they the same? Why not put the Address from below to the top and use it here. If it's different, then SecondaryAddress is a better name here.